data/presets.yaml

```diff
@@ -2101,6 +2101,71 @@ en:
         # 'area:highway=*'
         name: Road Surface
         terms: '<translate with synonyms or related terms for ''Road Surface'', separated by commas>'
+      attraction/amusement_ride:
+        # attraction=amusement_ride
+        name: Amusement Ride
+        # 'terms: theme park,carnival ride'
+        terms: '<translate with synonyms or related terms for ''Amusement Ride'', separated by commas>'
+      attraction/animal:
+        # attraction=animal
+        name: Animal
+        # 'terms: zoo,theme park,animal park,lion,tiger,bear'
+        terms: '<translate with synonyms or related terms for ''Animal'', separated by commas>'
+      attraction/big_wheel:
+        # attraction=big_wheel
+        name: Big Wheel
+        # 'terms: ferris wheel,theme park,amusement ride'
+        terms: '<translate with synonyms or related terms for ''Big Wheel'', separated by commas>'
+      attraction/bumper_car:
+        # attraction=bumper_car
+        name: Bumper Car
+        # 'terms: theme park,dodgem cars,autoscooter'
+        terms: '<translate with synonyms or related terms for ''Bumper Car'', separated by commas>'
+      attraction/bungee_jumping:
+        # attraction=bungee_jumping
+        name: Bungee Jumping
+        # 'terms: theme park,bungy jumping,jumping platform'
+        terms: '<translate with synonyms or related terms for ''Bungee Jumping'', separated by commas>'
+      attraction/carousel:
+        # attraction=carousel
```

Comment:
Part of the tagging proposal is an [`attraction=swing_carousel`](https://taginfo.openstreetmap.org/tags/attraction=swing_carousel) tag for [swing rides](https://en.wikipedia.org/wiki/Swing_ride). That caught my attention because “swing ride” apparently has a well-known translation in Vietnamese, but ordinary horse carousels don’t have as obvious a translation.